### ISSUE TITLE
in the section 5.1, after the training models converge, you continue to train them for 5000 iterations

### ISSUE THREAD

**Issue by lucky-dersan** [happynear/NormFace#9]
you continue train for 5000 iterations, the training set is LFW dataset or the original training phase dataset?
i am confused about the training dataset. can you explain to me?

**Comment by happynear** [comment]
I just want to say that I didn't cherry pick one best accuracy to report. The 5,000 iterations are still on CASIA-Webface dataset.
